src/Files.App/Utils/Storage/Operations/FileOperationsHelpers.cs

```diff
@@ -1018,5 +1018,27 @@ protected override void Dispose(bool disposing)
 				}
 			}
 		}
+
+		public static string GetIncrementalName(bool overWriteOnCopy, string? filePathToCheck, string? filePathToCopy)
+		{
+			if (filePathToCheck == null)
+				return null;			
+
+			if ((!Path.Exists(filePathToCheck)) || overWriteOnCopy || filePathToCheck == filePathToCopy)
+				return Path.GetFileName(filePathToCheck);
+
+			int i = 2;
+			string filePath = filePathToCheck;
+			if (Path.HasExtension(filePathToCheck))
+				filePath = filePathToCheck.Substring(0, filePathToCheck.LastIndexOf("."));
+
+			Func<int, string> genFilePath = x => string.Concat([filePath, " (", x.ToString(), ")", Path.GetExtension(filePathToCheck)]);
+
+			while (Path.HasExtension(filePathToCheck) ? File.Exists(genFilePath(i)) : Path.Exists(genFilePath(i)))
+			{
```

Comment:
1. Remove brackets and insert a line break right before `return` statement.
2. Let's use `index++;` instead.